frontend/src/metabase/nav/containers/Navbar.jsx

```diff
@@ -141,71 +118,12 @@ export default class Navbar extends Component {
             mx={1}
             hover={{ backgroundColor: getDefaultSearchColor() }}
           >
-            <LogoIconContainer>
-              <LogoIcon dark height={32} />
-            </LogoIconContainer>
+            <LogoIconContainer></LogoIconContainer>
```

Comment:
Should a logo or something be here? Currently, there is a clickable transparent square at the top left of the sidebar, please see:

![CleanShot 2022-03-23 at 17 38 05](https://user-images.githubusercontent.com/17258145/159762131-ace8a8d2-ef36-48f4-bc2e-7b191b5696a7.gif)